pkg/core/capability/caldera/caldera.go

```diff
@@ -48,6 +48,25 @@ func (capability *calderaCapability) GetType() string {
 	return capabilityName
 }
 
+// Cleanup will attempt to remove any artifacts created by SOARCA on the Caldera installation.
+// This function is idempotent. This function will only remove artifacts tied to SOARCA-related
+// Operation objects for Operations that are finished, to prevent race conditions. Cleanup may
+// fail silently if it encounters errors.
+func Cleanup(cc ICalderaConnection) {
```

Comment:
Are you intentionally making this public? If it only gets used in this file, make it lowercase to make it private.

Reply:
In the previous structure this was required for testing, but I can convert it to a private one now.